### DISCUSSION TITLE
Cat Collars Color

### DISCUSSION THREAD

**Discussion by itsMarkiS** [PurpurMC/Purpur#577] [__Completed__] [answered]
Well, since 1.14 cat and ocelot are different mobs and now cats have collars, just like wolfs
In purpur.yml we have a configuration option for wolfs aka **default-collar-color** so why not for the cats too?
a simple feature but consistent and more customization in purpur

**Comment by granny** [comment, marked as answer]
Implemented as of this commit https://github.com/pl3xgaming/Purpur/commit/158ae614632302766905ab06a68d83b47281e768
